db/seeds.rb

```diff
@@ -41,25 +41,27 @@ def make_lang_skills
 end
 
 User.role_collection.each do |role|
-  User.find_or_create_by(email: "#{role}@example.com") do |user|
+  user = User.find_or_create_by(email: "#{role}@example.com") do |user|
     user.password = 'asdfasdf'
     user.role = role
-    user.profile = Profile.new do |profile|
-      profile.build_contact(
-        first_name: Faker::Name.first_name,
-        last_name: Faker::Name.last_name,
-        postal_code: Faker::Address.zip_code,
-        city: Faker::Address.city,
-        street: Faker::Address.street_address,
-        primary_email: Faker::Internet.email,
-        primary_phone: Faker::PhoneNumber.phone_number
-      )
-      profile.profession = Faker::Company.profession
-      availability_collection.each do |availability|
-        profile[availability] = [true, false].sample
-      end
+  end
+  user.profile = Profile.new do |profile|
```

Comment:
Doesn't this just adds a profile only for the last user created? What is the reason for that?

Reply:
I don't know why, but it didn't work anymore as it was before, and changeing it this way got it running again.

Reply:
For unknown reason the way it was before it didn't work anymore, and changing it this way got it back running again... so ...